.gitignore

```diff
@@ -4,4 +4,3 @@ node_modules/
 coverage/
 .nyc_output/
 test-output.txt
-/.gtm/
```

Comment:
This should not be removed. gtm provides 2 git aliases to perform synchronization of the time statistics: `git fetchgtm` and `git pushgtm`.

By applying the githooks as follows, you can automate this:

pre-commit:
```sh
#!/bin/sh
git fetchgtm
```

post-commit:
```sh
#!/bin/sh
gtm commit --yes
git pushgtm
```

Reply:
thanks, iĺl make sure I have the hooks in each project

Reply:
Tip: first attempt to run `git fetchgtm` manually. If the project has no time metrics yet whatsoever, that will crash, preventing you from making the first commit containing time metrics.